roles/core/defaults/main.yml

```
@@ -5,5 +5,6 @@ core:
   ran_subnet: "" # set it to empty to get subnet from 'data_iface'
 
   helm:
+    local_charts: false
     chart_ref: aether/sd-core
     chart_version: 0.12.6
```

Update version here:
```suggestion
    chart_version: 1.0.4
```